### PR TITLE
Submission for SW carpentry instructor training

### PR DESCRIPTION
**Pull request by maglet**
I have edited the create.md document to add an exercise at the end for
credit the software carpentry trainer certification. This exercise asks
the student to replicate a file structure and remove data files in the
context of starting a new project with similar data.

### CODE REVIEW

**Comment by gvwilson** [comment]
I like this exercise - can you please re-do the PR without `.Rhistory` and the `.html` file and then we'll merge?

**Comment by maglet** [comment]
Great! Changes made

C. Tobin Magle, PhD
Bioinformationist
Senior Instructor
Health Sciences Library
University of Colorado Anschutz Medical Campus
http://orcid.org/0000-0003-3185-7034
http://hslibrary.ucdenver.edu<http://hslibrary.ucdenver.edu/>
tobin.magle@ucdenver.edu<mailto:tobin.magle@ucdenver.edu>
303-724-2114

Ask Us! - 303-724-2152 or http://hslibrary.ucdenver.edu/aal/
Like us on Facebook<https://www.facebook.com/UCDenverHSLibrary> | Follow us on Twitter<https://twitter.com/UCDenverHSL> #HSLAnschutz | https://www.pinterest.com/HSLAnschutz/ | http://instagram.com/HSLAnschutz,

File: 02-create.md [hunk 1]
```diff
@@ -432,3 +432,37 @@ but it does find the copy in `thesis` that we didn't delete.
...
+>
+> Use the `file-structure` directory under `shell-data` as an example.
+> The goal is to copy the file structure of the `2016-5-18-data` folder
+> into a folder called `2016-5-20-data` and remove the data files from 
+> the first folder.
+> 
+> Navigate to the `file-structure` folder. Which of the following set of 
+> commands would achieve this objective? What would the other commands do?
+>
+>#1
```
Comment (shwina) [inline 02-create.md]: The `#1` gets rendered as a heading. See here: https://github.com/maglet/shell-novice/blob/f59bb3e9a3c05ed2a7f296f44f667cfe1835d718/02-create.md.

Just use `1.`, `2.`, and `3.` instead?

**Comment by gdevenyi** [comment]
Thanks for your contribution. The repo has diverged significantly from your changes.

If you have time, can you please re-create these PRs addressing the requested changes and not including the extraneous files from the start.

**Pull request closed without merging**